### ISSUE TITLE
ettercap

### ISSUE THREAD

**Issue by unikquentin** [Ettercap/ettercap#1142]

**Comment by koeppea** [comment]
@unikquentin when you're in the empty "_build_" directory, you have to append "`..`" to the end of the `cmake` command, to instruct cmake where to look for `CMakeLists.txt `files.

**Comment by unikquentin** [comment]
Sorry, in the previous message i forgot to put the ".." . I already put it at the end of the cmake. 
When i enter the command : cmake -DCMAKE_BUILD_TYPE=Debug -DENABLE_IPV6=ON -DOPENSSL_ROOTDIR='brew --prefix openssl' ..
I have this error : 
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Warning (dev) at /usr/local/Cellar/cmake/3.21.2/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (GTK3_GTK)
  does not match the name of the calling package (GTK3).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/Modules/FindGTK3.cmake:546 (find_package_handle_standard_args)
  cmake/Modules/EttercapLibCheck.cmake:48 (find_package)
  CMakeLists.txt:106 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at /usr/local/Cellar/cmake/3.21.2/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
  system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY
  OPENSSL_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.21.2/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /usr/local/Cellar/cmake/3.21.2/share/cmake/Modules/FindOpenSSL.cmake:574 (find_package_handle_standard_args)
  cmake/Modules/EttercapLibCheck.cmake:97 (find_package)
  CMakeLists.txt:106 (include)


-- Configuring incomplete, errors occurred!
See also "/Users/"myname"/Applications/ettercap/build/CMakeFiles/CMakeOutput.log".
See also "/Users/"myname"/Applications/ettercap/build/CMakeFiles/CMakeError.log".

If you can help me that will be great, i m knew with homebrew and ettercap so i don't understand all the errors. I try to check on web but i found nothing.

Unik

**Comment by koeppea** [comment]
This is the actual error:

>  Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY

Do you have OpenSSL installed?
What it the output of the command `brew --prefix openssl`?

**Comment by unikquentin** [comment]
Yes, i think. I try to install openssl and i have this message : 
"Warning: openssl@1.1 1.1.1l is already installed and up-to-date."

I don't understand what you mean by the "output of the command brew --prefix openssl?"

When i installed ettercap, i done this : 
brew reinstall check curl cmake ghostscript gtk+ gtk+3 libnet rtmpdump geoip  
 rm -rf etterczp 
git clone https://github.com/Ettercap/ettercap
cd ettercap
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Debug -DENABLE_IPV6=ON -DOPENSSL_ROOTDIR='brew --prefix openssl' .." 
When i do a sniffing, Ettercap freeze (he stops running)

**Comment by koeppea** [comment]
Ah so you are able to build ettercap but it freezes when running the application.

The freeze of ettercap on MacOSX is a known issue but unfortunately for which we don’t have a solution yet.

**Comment by unikquentin** [comment]
Yes i have  ettercap 
Okay, i believed it was because the cmake didn't worked 

thanks for helping me
